pandas/core/series.py

```diff
@@ -1462,63 +1463,21 @@ def dot(self, other):
         else:  # pragma: no cover
             raise TypeError('unsupported type: %s' % type(other))
 
-    def searchsorted(self, v, side='left', sorter=None):
-        """Find indices where elements should be inserted to maintain order.
+    @Substitution(klass='Series')
+    @Appender(base._shared_docs['searchsorted'])
+    def searchsorted(self, v, side='left', *args):
+        validate_args(args, min_length=0, max_length=1)
+        sorter = None if not args else args[0]
 
-        Find the indices into a sorted Series `self` such that, if the
-        corresponding elements in `v` were inserted before the indices, the
-        order of `self` would be preserved.
+        if not isinstance(self._values, Index):
+            if sorter is not None:
+                sorter = com._ensure_platform_int(sorter)
 
```

Comment:
> That is the way of bugs. If we're going to add sorter but not accept it as a real argument, we need to verify that sorter is None or raise an error.

I agree that we should raise an error in case of ignoring the keyword. 
But for that reason, it's actually much simpler of just passing it through to numpy, then we don't have to care about it further.

I don't see a compelling reason to start deprecating it, so my preference is the first bullet point as well, or the third.

So